activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/ProcessModelValidatorConfiguration.java

```diff
@@ -166,6 +167,15 @@ public BpmnModelServiceTaskImplementationValidator bpmnModelServiceTaskImplement
                                                                connectorModelContentConverter, flowElementsExtractor);
     }
 
+    @Bean
+    @ConditionalOnMissingBean
+    public BpmnModelServiceTaskCatchBoundaryValidator bpmnModelServiceTaskBoundaryValidator(FlowElementsExtractor flowElementsExtractor) {
+        ServiceTaskImplementationType[] serviceTaskImplementationTypes = new ServiceTaskImplementationType[] {
```

Comment:
Any reason to use an array instead of a list?

Reply:
Changed to list. #947